virtcontainers/container.go

```diff
@@ -640,6 +641,7 @@ func newContainer(sandbox *Sandbox, contConfig ContainerConfig) (*Container, err
 		runPath:       store.ContainerRuntimeRootPath(sandbox.id, contConfig.ID),
 		configPath:    store.ContainerConfigurationRootPath(sandbox.id, contConfig.ID),
 		containerPath: filepath.Join(sandbox.id, contConfig.ID),
+		rootfsSuffix:  "rootfs",
```

Comment:
@mcastelino @ganeshmaharaj This works for us since the storage drivers provide the root file system under "rootfs" path. We talked about making this generic, so maybe we can deduce the suffix at runtime by looking at the difference between the rootfs mountpoint and the rootfs path provided in the OCI spec. 
I am fine if you want to address this in a separate PR.

Reply:
@amshinde if it can be a seperate PR, that would be good. Will also give us time to look through the code and clean-up anything else we might have missed.